rholang/src/main/scala/coop/rchain/rholang/interpreter/accounting/package.scala

```diff
@@ -1,3 +1,23 @@
 package coop.rchain.rholang.interpreter
 
-package object accounting extends Costs
+import cats.Monad
+import cats.implicits._
+import cats.mtl.MonadState
+
+import coop.rchain.rholang.interpreter.errors.OutOfPhlogistonsError
+
+package object accounting extends Costs {
+
+  type _cost[F[_]] = MonadState[F, Cost]
```

Comment:
consider using `Cell` (with `flatModify`, without `set`) in your next PR

Reply:
I really don't get why if MonadState suffices. `set` is a direct consequence of `modify`, which cell also has. Can you elaborate on the reasons for using Cell?